packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -921,29 +921,30 @@ const StepTitle = styled.span< StepTitleProps & HTMLAttributes< HTMLSpanElement
 			? props.theme.colors.textColorDark
 			: props.theme.colors.textColorDisabled };
 	font-weight: ${ ( props ) => props.theme.weights.bold };
-	margin-right: ${ ( props ) => ( props.fullWidth ? '0' : '8px' ) };
 	flex: 1;
-
-	.rtl & {
-		margin-right: 0;
-		margin-left: ${ ( props ) => ( props.fullWidth ? '0' : '8px' ) };
-	}
+	margin-right: 8px;
 `;
 
 interface StepTitleProps {
 	isComplete?: boolean;
 	isActive?: boolean;
-	fullWidth?: boolean;
 }
 
-const StepHeader = styled.h2< StepHeaderProps & HTMLAttributes< HTMLHeadingElement > >`
-	font-size: 20px;
+const StepHeaderWrapper = styled.div< StepHeaderProps & HTMLAttributes< HTMLDivElement > >`
 	display: flex;
 	width: 100%;
 	align-items: center;
 	margin: 0 0 ${ ( props ) => ( props.isComplete || props.isActive ? '8px' : '0' ) };
 `;
 
+const StepHeader = styled.h2< StepHeaderProps & HTMLAttributes< HTMLHeadingElement > >`
+	font-size: 20px;
+	display: flex;
+	align-items: center;
+	margin: 0;
+	flex: 1;
```

Comment:
The `fullWidth` [removed here](https://github.com/Automattic/wp-calypso/pull/104279/files#diff-756c6ffb6eecc8ea890046c286928cd2631d88b9ee52e9c3157da29f17240b47L924) is not required anymore as it is replaced by `flex: 1`